### ISSUE TITLE
In Tensorflow code: last_h(last item embedding) instead of last (last_h * w1) when concatenate with Sg 

### ISSUE THREAD

**Issue by priyanka15047** [CRIPAC-DIG/SR-GNN#14]
Tensorflow code/model.py line no. 46
ma = tf.concat([tf.reduce_sum(tf.reshape(coef, [self.batch_size, -1, 1]) * seq_h, 1),                            tf.reshape(last, [-1, self.out_size])], -1)
I think 'last' should be 'last_h' in above line, as in code:
last_h = tf.gather_nd(re_embedding, tf.stack([tf.range(self.batch_size), last_id], axis=1))
last = tf.matmul(last_h, self.nasr_w1)
According to paper Sg should be concatenate with last item embedding. 
Please clarify.

**Comment by WJSAI** [comment]
I found the same problem, I think maybe your point is right.
